client/app/components/calendar/RoomFilters.jsx

```diff
@@ -0,0 +1,34 @@
+import React  from 'react'
+
+import { Checkbox, Label } from 'react-bootstrap'
+
+import './filters.scss'
+
+export default class RoomFilters extends React.Component {
+    
+    _handleCheckbox(event){
+        if(!event.target.checked){
+            this.props.add(parseInt(event.target.value));
+        }
+        else{
+            this.props.delete(parseInt(event.target.value));
+        }
+    }
+
+    render(){
+        let filters = this.props.conferenceRooms.map(
+            (conferenceRoom) => {
+                return (
+                    <div className="filter-box" style={{backgroundColor: conferenceRoom.color}}>
```

Comment:
Each child in an array or iterator should have a unique "key" prop.
Listen to Reacts warnings plox